mathics/builtin/graphics.py

```diff
@@ -1087,6 +1094,75 @@ def stylebox_style(style, specs):
                     raise BoxExpressionError
             return new_style
 
+        failed = []
+
+        def build_error_box1(style):
```

Comment:
If we agree that setting the background is the right thing, these two functions can be safely removed.

Reply:
I agree that setting the background is the right thing.